package.json

```diff
@@ -41,6 +41,7 @@
     "prettier": "^2.7.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "storybook-addon-pseudo-states": "^1.15.1",
```

Comment:
Nit: I think this should be a devDepenency of the `docs` package.  (In a mono-repo, try to include dependencies directly in the workspaces they are needed unless they are required at the top level for somer reason, e.g., build/lint commands.)